src/CattyTests/Project/ProjectTests.swift

```diff
@@ -215,7 +214,7 @@ class ProjectsTest: XCTestCase {
 
         let copiedSetVariableBrick = copiedScript.brickList[0] as! SetVariableBrick
         XCTAssertNotNil(copiedSetVariableBrick.userVariable)
-        XCTAssertNotEqual(variable, copiedSetVariableBrick.userVariable)
+        XCTAssertFalse(variable === copiedSetVariableBrick.userVariable)
```

Comment:
Thanks!

Reply:
Upon removing the default initialiser, there's a runtime exception when testing  in ProjectParser.m at line 136, which is
`[[NSClassFromString(classname) alloc ] init];` and causes an exception when classname is equal to "UserVariable" and tries to call no parameter initialiser of UserVariable. So currently I chose not to remove it. Please let me know if anything better can be done here.

Reply:
Good point. So we have to keep the default initializer until the deprecated `ProjectParser` has been removed. Thanks for pointing that out, I will create a separate ticket for this. Also that implies that we can not make `name` `let` which I was planning to do. However, to prevent the usage of this initializer in Swift, please set it to private (i.e., `override private init()`). This will work due to your `@objcMembers` which is exposing all methods to Objective C.